packages/satori-js/client.ts

```diff
@@ -270,4 +270,16 @@ export class Client {
       return Promise.resolve(response !== undefined);
     });
   }
+
+    /** Delete an Identity. */
+    async deleteIdentify(session: Session, id: string) {
```

Comment:
should the function name be `deleteIdentify` or `deleteIdentity`?

Reply:
I think you are looking at an outdated version of the PR

Reply:
the commit that handles that incoherence is already here afc3c92974f11420e815f32a09a730a361afbf0c